pkg/crc/image/image.go

```diff
@@ -95,39 +112,63 @@ func getImageName(preset crcpreset.Preset) string {
 	}
 }
 
-func PullBundle(preset crcpreset.Preset) error {
+func getPresetNameE(imageName string) (crcpreset.Preset, error) {
```

Comment:
`getPresetNameE`; What does the `E` suffix mean here?

Reply:
@gbraad `E` means with error. This naming format is heavily used in spf13/cast/caste.go and also some of our codebase.